niimpy/location.py

```diff
@@ -92,7 +101,7 @@ def filter_location(location,
 def bin_location(location,
                  bin_width=10,
                  aggregation='median',
-                 columns_to_aggregate=['double_latitude', 'double_longitude']):
+                 columns_to_aggregate=None):
```

Comment:
add doc for the other parameters

Reply:
They all have docs

Reply:
oh yeah you are right, sorry, I didn't see the rest of the function.